include/open62541/client.h

```diff
@@ -147,6 +147,10 @@ typedef struct {
     /* Certificate Verification Plugin */
     UA_CertificateVerification certificateVerification;
 
+    /* Available SecurityPolicies for Authentication. */
+    size_t authSecurityPoliciesSize;
```

Comment:
I think @mlgiraud said that the same certificates aught to be used for SecureChannel and authentication.
Please check with him.
If that is the case, then we don't need to configure separate authSecurityPoliciesSize.